### PR TITLE
fix: text node bindings with 'null' or 'undefined' updates

### PR DESCRIPTION
**Pull request by benouat**

### CODE REVIEW

File: src/iv/instructions.ts [hunk 1]
```diff
@@ -256,23 +253,23 @@ export function textCreate(value: any) {
...
  * @param value Value to write. This value will go through stringification.
  */
 export function textCreateBound(value: any): void {
   textCreate(value);
-  if (!creationMode) {
-    const node = cursor as IvText;
-    if (node.value !== value) {
-      (node.native as Text).textContent = node.value = value;
+  const node = cursor as IvText;
+  const valueAsString = stringify(value);
```
Comment (mhevery) [inline src/iv/instructions.ts]: We should not run the `stringify` function until after we check that it is different. If we do this here than we will be creating memory pressure every time this runs here.

File: src/iv/instructions.ts [hunk 2]
```diff
@@ -256,23 +253,23 @@ export function textCreate(value: any) {
...
+  if (creationMode) {
+    node.value = valueAsString;
+  } else {
+    if (node.value !== valueAsString) {
+      (node.native as Text).textContent = node.value = valueAsString;
```
Comment (mhevery) [inline src/iv/instructions.ts]: change to: `(node.native as Text).textContent = stringify(node.value = valueAsString);`

**Comment by mhevery** [comment]
Also please rebase on latest

**Comment by benouat** [comment]
@mhevery branch has been rebased on latest, and changes have been made accordingly to your review.

To make it work properly, I also had to make the cursor to be "consumed", even for static text node when executed in update mode.

Nonetheless, `textCreateBound(value: any)` should be updated to handle concatenation of both static text and expressions, a bit in the same way you did in #9 or with any other solution if we find a different/better one.

**Comment by benouat** [comment]
Closing this PR as #14 has basically fixed the issue.

**Pull request closed without merging**